test/threadstest.c

```diff
@@ -436,6 +436,7 @@ static void thread_shared_evp_pkey(void)
         if (!TEST_ptr(ctx))
             goto err;
 
+        ptlen = sizeof(ptbuf);
```

Comment:
Perhaps also move the declaration inside the for loop? Or drop the initial assignment at the declaration as it is useless.

Reply:
done, thank u for ur reminder.